src/basis.jl

```diff
@@ -48,8 +48,14 @@ notion of degree is specified by further parameters) (mandatory)
 * `r0 = 2.5` : rough estimate for nearest neighbour distance
 * `rcut = 5.0`: outer cuttoff, Å 
 * `rin = 0.0`: inner cuttoff, Å 
-* `pcut = 2`: outer cutoff parameter
+* `pcut = 2`: outer cutoff parameter; 
+      * `pcut=2`: function forced to smoothly go to zero at the outer cutoff 
```

Comment:
I think for pcut=2 it's worth being precise ("function and first derivative go to zero ..." if that's correct). Sorry, I know I was the cause of these changes! It was more the "zeroth derivative" stuff that I objected to

Reply:
of course and thanks - it's a bit hard for me to tell what would be the clearest.